apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -416,7 +416,7 @@ export function PublishModal({
             entities={entities}
             path={path}
             files={files}
-            containerClassNames="px-3 py-4 md:px-5"
+            containerClassNames="px-3 py-4 md:px-5 overflow-auto"
```

Comment:
I don't think that we want to allow horizontal scroll
```suggestion
            containerClassNames="px-3 py-4 md:px-5 overflow-y-auto"
```